schemas/layer-4.cue

```diff
@@ -1,35 +1,45 @@
 // Top level schema //
-
-"evaluation-plans": [...#EvaluationPlan]
+#FrameworkEvaluation: {
+    // name of the framework being evaluated
+    name: string
+    // ID of the framework being evaluated
+    frameworkID: string
+    // one or more evaluations of the framework controls
+    evaluations: [#ControlEvaluation, ...]
+}
 
 // Types
 
+#URL: =~"^https?://[^\\s]+$"
+
 #ControlEvaluation: {
+    // name of the control being evaluated
     name: string
-    "control-id": string
-    result: #Result
-    message: string
-    "documentation-url"?: =~"^https?://[^\\s]+$"
```

Comment:
maybe be intended for "planned use"

Reply:
What would the content at the value of `documentation-url` describe/document? As it is optional, until we have more clarity there, I propose we remove it.